internal/query/photo_count_per_month.go

```diff
@@ -0,0 +1,24 @@
+package query
+
+// PhotoCountResult contains photos per month
+type PhotoCountResult struct {
+	PhotoYear  int
+	PhotoMonth int
+	Count      int
+}
+
+// GetPhotoCountPerMonth couts photos per month and year
+func (s *Repo) GetPhotoCountPerMonth() (results []PhotoCountResult, err error) {
+	q := s.db.NewScope(nil).DB()
+
+	q = q.Table("photos").
+		Select("photos.photo_year, photos.photo_month, COUNT(*) AS count").
+		Group("photos.photo_year, photos.photo_month").
+		Order("photos.photo_year DESC, photos.photo_month DESC")
```

Comment:
Looks good, but you might want to ignore deleted / archived photos for this?

Reply:
:heavy_check_mark: